packages/bpk-component-drawer/src/BpkDrawerContent-test.tsx

```diff
@@ -50,7 +50,7 @@ describe('BpkDrawerContent', () => {
     const { asFragment } = render(
       <BpkDrawerContent
         id="my-drawer"
-        className="my-classname"
+        additionalClassName="my-classname"
```

Comment:
Would it be possible to keep it as `className`, otherwise it will be a breaking change for consumers

Reply:
I get an error saying that className is a banned
`Prop className is forbidden. See https://github.com/Skyscanner/eslint-plugin-rules?tab=readme-ov-file#forbid-component-props`

This shouldn't impact consumers as the value is passed from BpkDrawer, which hasn't changed its prop names?

Reply:
Interesting - thanks for sharing this!

Reply:
`BpkDrawerContent` should also be an internal component, right? Not exposed to the consumers that are using just `BpkDrawer`?